### DISCUSSION TITLE
Como testar seu "nivel" em flutter?

### DISCUSSION THREAD

**Discussion by Developer-jdantas** [Flutterando/forum#153] [Perguntas e Respostas]
Bom dia.
Sempre tenho dúvidas sobre "o nível" de programador em flutter que estou no momento. Quais os desafios que a galera do flutterando propõem para podermos ter uma noção disso, bem como para encontrar nossas fraquezas e aperfeiçoa-las?
Obrigado!

**Comment by codetrindade** [comment]
Em geral, sua experiência anterior em Flutter determinará o nível de habilidade que você pode se enquadrar. 
Existem alguns testes de aptidão online específicos em Flutter que você pode fazer para testar seu conhecimento em Flutter.

**Alguns sites para testar** 
https://welovedevs.com/app/tests/flutter
https://www.testdome.com/tests/flutter-online-test/231
https://skillvalue.com/en/quiz/mobile/flutter-quiz-medium-level/
https://testlify.com/test-library/flutter/
